.travis.yml

```diff
@@ -0,0 +1,41 @@
+language: java
+jdk: openjdk8
+sudo: required
+dist: trusty
+
+install:
+- wget http://www-us.apache.org/dist/maven/maven-3/3.3.9/binaries/apache-maven-3.3.9-bin.tar.gz
+- tar xzf apache-maven-3.3.9-bin.tar.gz
+- curl -s -L https://github.com/mozilla/smarthome/releases/latest | egrep -o /mozilla/smarthome/releases/download/.*/m2repository.tar.gz | wget --base=http://github.com/ -i -
+- tar xf m2repository.tar.gz -C $HOME/.m2/repository/
+
+before_script:
+- rm $HOME/.m2/settings.xml
+- export MAVEN_OPTS="-Dmaven.artifact.threads=3 -Xmx2469m -XX:MaxPermSize=512m"
+
```

Comment:
So are we sure that the bundles

- `bundles/automation/org.eclipse.smarthome.automation.module.timer.test`
- `bundles/automation/org.eclipse.smarthome.automation.module.script.test`

don't fail intermittently? Their build exclusion has been removed from the `PROJECTS` env variable.